contrib/python-sambacc.spec

```diff
@@ -0,0 +1,66 @@
+%global bname sambacc
+# set xversion to define the default version number
+%define xversion 0.1
+# set pversion for a customized python package version string
+%{?!pversion: %define pversion %{xversion}}
+# set rversion for a customized rpm version
+%{?!rversion: %define rversion %{xversion}}
+
+
+Name:           python-%{bname}
+Version:        %{rversion}
+Release:        1%{?dist}
+Summary:        Samba Container Configurator
```

Comment:
Current description seems more like `Summary`. We could have more explanatory para for `%description` down the line.

Reply:
See my suggestion on new changes.